ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -16,6 +17,13 @@ expiration_date=$(date -d '8 hours' --iso=minutes --utc)
 function join_by { local IFS="$1"; shift; echo "$*"; }
 
 REGION="${LEASED_RESOURCE}"
+
+aws_region2="${REGION}"
```

Comment:
`aws_region2` is not a good variable name. It would be better to pick a variable name that describes why we need this second region.

Reply:
Updated.